### ISSUE TITLE
[BUG - Release]

### ISSUE THREAD

**Issue by Luluqhge** [UE4SS-RE/RE-UE4SS#593]
3.0.1

Palworld

Game crashes upon loading the world

1. Have only UE4SS 3.0.1 installed
2. Launch Game
3. Load into a world
4. upon loading game crashes

I would expect to have the game run fine with only this installed but game crashes, was fine before but now it does this, and when I do delete all the files in this mod, game runs fine.

 - OS: Win11

As I said above, was fine a few hours ago, but now the game crashes every time I load a world, and only works if UE4SS is uninstalled.

**Comment by UE4SS** [comment]
Use the issue template, and provide all the information asked in it.
In the meantime, you could try the experimental build by going [here](https://github.com/UE4SS-RE/RE-UE4SS/releases/tag/experimental) and scrolling down until you find an asset with a recent date, ideally the most recent date.

**Comment by Luluqhge** [comment]
This is my first time using GitHub so I'm sorry for not understanding
exactly what the issue template was asking of me. But I was able to figure
out what was wrong, I will update my post when I am able, I changed
something in the settings file as per instruction from a mod and it worked.
But what I don't understand is that the game was working fine with the mod
before but all of a sudden stopped working and then worked when I changed
one of the settings.
